src/m_obj.c

```diff
@@ -919,6 +920,35 @@ int obj_issignaloutlet(const t_object *x, int m)
     "MAINSIGNALIN" has been provided, in which case the object won't
     promote scalars correctly.  Nonetheless we provide it so that at least
     such a badly written object won't crash Pd. */
+
+void obj_sigoutletsetchans(const t_object *x, int m, int nchans)
+{
+    int n;
+    t_outlet *o2;
+    for (o2 = x->ob_outlet, n = 0; o2 && m--; o2 = o2->o_next);
+    if (o2 && (o2->o_sym == &s_signal))
+    {
+        o2->o_nchans = nchans;
+    }
+}
+
+int obj_sigoutletgetchans(const t_object *x, int m)
+{
+    int n;
+    t_outlet *o2;
+    for (o2 = x->ob_outlet, n = 0; o2 && m--; o2 = o2->o_next);
+
+    if (o2 && (o2->o_sym == &s_signal)){
+        if (o2->o_nchans == 0){
+            return 1;
+        }
+        return o2->o_nchans;
+    }
+    else{
+        return 0;
+    }
```

Comment:
Again, code style. Opening braces go on their own line. Omit braces for if-statements with a single expressions (also above in `obj_sigoutletsetchans`).

Reply:
```
if (o2->o_nchans == 0){
    return 1;
}
return o2->o_nchans;
```
I think the following is more natural:
```
if (o2->o_nchans == 0)
    return 1;
else
    return o2->o_nchans;
```
Or even:
```
return (o2->o_nchans > 0) ? o2->o_nchans : 1;
```